From d27b846f797d2cab3d27f8cf64ed9648c692293d Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@freescale.com>
Date: Tue, 18 May 2010 20:00:30 +0800
Subject: [PATCH] ENGR00123436 Clear GPMI LOCK_CS to lower suspend current.

GPMI LOCK_CS set to low can lower suspend current about 5mA.

Signed-off-by: Frank Li <Frank.Li@freescale.com>
Signed-off-by: Turley Patrick <b18601@freescale.com>
---
 drivers/mtd/nand/gpmi-nfc/gpmi-nfc-hal-v0.c |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/drivers/mtd/nand/gpmi-nfc/gpmi-nfc-hal-v0.c b/drivers/mtd/nand/gpmi-nfc/gpmi-nfc-hal-v0.c
index fddccaf..f80e1c9 100644
--- a/drivers/mtd/nand/gpmi-nfc/gpmi-nfc-hal-v0.c
+++ b/drivers/mtd/nand/gpmi-nfc/gpmi-nfc-hal-v0.c
@@ -292,7 +292,6 @@ static int send_command(struct gpmi_nfc_data *this, unsigned chip,
 	(*d)->cmd.address = buffer;
 
 	(*d)->cmd.pio_words[0] =
-		BM_GPMI_CTRL0_LOCK_CS                    |
 		BF_GPMI_CTRL0_COMMAND_MODE(command_mode) |
 		BM_GPMI_CTRL0_WORD_LENGTH                |
 		BF_GPMI_CTRL0_CS(chip)                   |
@@ -364,7 +363,6 @@ static int send_data(struct gpmi_nfc_data *this, unsigned chip,
 	(*d)->cmd.address = buffer;
 
 	(*d)->cmd.pio_words[0] =
-		BM_GPMI_CTRL0_LOCK_CS                    |
 		BF_GPMI_CTRL0_COMMAND_MODE(command_mode) |
 		BM_GPMI_CTRL0_WORD_LENGTH                |
 		BF_GPMI_CTRL0_CS(chip)                   |
@@ -435,7 +433,6 @@ static int read_data(struct gpmi_nfc_data *this, unsigned chip,
 	(*d)->cmd.address = buffer;
 
 	(*d)->cmd.pio_words[0] =
-		BM_GPMI_CTRL0_LOCK_CS                    |
 		BF_GPMI_CTRL0_COMMAND_MODE(command_mode) |
 		BM_GPMI_CTRL0_WORD_LENGTH                |
 		BF_GPMI_CTRL0_CS(chip)                   |
@@ -474,7 +471,6 @@ static int read_data(struct gpmi_nfc_data *this, unsigned chip,
 
 	(*d)->cmd.pio_words[0] =
 		BF_GPMI_CTRL0_COMMAND_MODE(command_mode) |
-		BM_GPMI_CTRL0_LOCK_CS                    |
 		BM_GPMI_CTRL0_WORD_LENGTH                |
 		BF_GPMI_CTRL0_CS(chip)                   |
 		BF_GPMI_CTRL0_ADDRESS(address)           |
@@ -550,7 +546,6 @@ static int send_page(struct gpmi_nfc_data *this, unsigned chip,
 	(*d)->cmd.address = 0;
 
 	(*d)->cmd.pio_words[0] =
-		BM_GPMI_CTRL0_LOCK_CS                    |
 		BF_GPMI_CTRL0_COMMAND_MODE(command_mode) |
 		BM_GPMI_CTRL0_WORD_LENGTH                |
 		BF_GPMI_CTRL0_CS(chip)                   |
@@ -641,7 +636,6 @@ static int read_page(struct gpmi_nfc_data *this, unsigned chip,
 
 	(*d)->cmd.pio_words[0] =
 		BF_GPMI_CTRL0_COMMAND_MODE(command_mode) |
-		BM_GPMI_CTRL0_LOCK_CS                    |
 		BM_GPMI_CTRL0_WORD_LENGTH                |
 		BF_GPMI_CTRL0_CS(chip)                   |
 		BF_GPMI_CTRL0_ADDRESS(address)           |
@@ -674,7 +668,6 @@ static int read_page(struct gpmi_nfc_data *this, unsigned chip,
 	(*d)->cmd.address = 0;
 
 	(*d)->cmd.pio_words[0] =
-		BM_GPMI_CTRL0_LOCK_CS                                 |
 		BF_GPMI_CTRL0_COMMAND_MODE(command_mode)              |
 		BM_GPMI_CTRL0_WORD_LENGTH                             |
 		BF_GPMI_CTRL0_CS(chip)                                |
@@ -715,7 +708,6 @@ static int read_page(struct gpmi_nfc_data *this, unsigned chip,
 
 	(*d)->cmd.pio_words[0] =
 		BF_GPMI_CTRL0_COMMAND_MODE(command_mode)              |
-		BM_GPMI_CTRL0_LOCK_CS                                 |
 		BM_GPMI_CTRL0_WORD_LENGTH                             |
 		BF_GPMI_CTRL0_CS(chip)                                |
 		BF_GPMI_CTRL0_ADDRESS(address)                        |
-- 
1.5.4.4

